SecretSanta/src/SecretSanta.Business/Gift.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace SecretSanta.Business
+{
+    public class Gift
+    {
+        public int Id { get; }
+        public string Title { get; set; }
```

Comment:
Might want to convert your non-readonly auto-properties to full properties. This way you can handle someone setting them null or trying to pass null into the constructor without having to write duplicate code. You may want to do this for your User class as well. I set _Title to "invalid" because the analyzers will start squawking if it is not initialized (even though it will be initialized no matter what).

```suggestion
        private string _Title = "invalid";
        public string Title 
        {
              get => _Title; 
              set => _Title = value ?? throw new ArgumentNullException(nameof(value)); 
        }
```

Reply:
Thank you for pointing that out!
I changed up my code thanks to the suggestion - but opted to null check in the constructor instead: 

``` c#
        public Gift(int id, string title, string description, string url, User user)
        {
            Id = id;
            Title = title ?? throw new ArgumentNullException(nameof(title));
            Description = description ?? throw new ArgumentNullException(nameof(description));
            Url = url ?? throw new ArgumentNullException(nameof(url));
            User = user ?? throw new ArgumentNullException(nameof(user));
        }
```